bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+    if( bindingConfig == null || bindingConfig.length() < 1 ) {
+        logger.warn("No settings configured for Hue binding for item '"+item+"'.");
+        return;
+    }
```

Comment:
Hey John, thanks.  Sure, parameterization, no problem.

However, my searches of the code base and the wiki have not helped me find the [referenced formatter](https://github.com/openhab/openhab/wiki/How-To-Contribute#code-style--conventions) or any documented coding style.

Any pointers would be appreciated.